ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -1576,26 +1578,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-confidential-trustedlaunch
-- as: azure-ipi-disc-oidc-oc-mirror-f28-destructive
-  cron: 5 11 15 * *
+- as: azure-ipi-disc-oidc-oc-mirror-mini-perm-f28-destructive
+  cron: 6 13 27 * *
   steps:
     cluster_profile: azure-qe
     env:
+      AZURE_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
       E2E_RUN_TAGS: '@disconnected'
+      ENABLE_MIN_PERMISSION_FOR_STS: "true"
       EXTRACT_MANIFEST_INCLUDED: "true"
       MIRROR_BIN: oc-mirror
       TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;Serial;Disruptive;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-disconnected-cco-manual-workload-identity
-- as: azure-ipi-disc-oidc-f14
-  cron: 1 15 14,28 * *
+- as: azure-ipi-disc-oidc-mini-perm-f14
```

Comment:
I guess we only need to enable one of a pair of specific config ci jobs.

Reply:
Oh, I saw AWS enable mini-perm for both destructive and non-destructive ci jobs for a specific config. If that, we can follow the same policy, ignore my above comments. cc @yunjiang29 and @gpei

Reply:
yes, I remember the rule we followed before is  if the job name won't exceed the length limit, we'll enable "mini-perm" to it.

Reply:
right, both destructive and non-destructive ci jobs enables mini-perms normally. But if the job name exceeds the limitation, only enable mini-perms on non-destructive ci job.

Reply:
> if the job name won't exceed the length limit, we'll enable "mini-perm" to it.

+1

Reply:
But I didn't enable mini-perms on all jobs where job name does not exceed the limitation, for example, sno/rhel workers/ingress-controller/mixarch/longduration...etc. Seems those jobs are also not enabled mini-perms on AWS.
Most updated jobs in this PR are related with installer features.